nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -12,9 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <string>
+#include <memory>
+#include <set>
```

Comment:
Are we using std::set in this class?

Reply:
No, it's not. It's to get uncrustify to be quiet. @orduno @crdelsey @mkhansen-intel @SteveMacenski Do you guys have a better way to work around uncrustify false positives? I don't really want to put something in the code that is uglier than simply including a header.